web/static/js/reducers/questionnaire.js

```diff
@@ -481,7 +481,7 @@ type ValidationState = {
 };
 
 const validateReducer = (reducer) => {
-  return (state: ValidationState, action: any) => {
+  return (state: ?ValidationState, action: any) => {
```

Comment:
Add a comment saying: "React will call this with an `undefined` the first time for initialization. We mimic that in the specs, so ValidationState needs to become optional here."